### DISCUSSION TITLE
Anime4k Shaders

### DISCUSSION THREAD

**Discussion by ramonsantana** [mpvnet-player/mpv.net#498] [Q&A]
How do i add Anime4k shaders to this version of MPV?
On regular mpv it goes into the shaders folder

**Comment by hooke007** [comment]
There is no rule that you must put shaders into somewhere.
https://mpv.io/manual/master/#options-glsl-shaders

`glsl-shaders=<path-to-any-place-you-like>`

If you are searching for the equivalents of https://mpv.io/manual/master/#configuration-files
It is doc here. https://github.com/mpvnet-player/mpv.net/blob/master/docs/Manual.md#settings
